### ISSUE TITLE
More info about required modules

### ISSUE THREAD

**Issue by hirale** [DegrizNet/OpenMageTheme#1]
First, I want to thank you for this awesome project!

I noticed that to unlock the full potential of the theme, it mentions the need for the following modules:

- KSv Banner
- Mana Filters
- J2T AddToCart
- Degriz Modules

However, I wasn't able to find links or any guidance on where to download or install these required modules. It would be great if you could either provide links to these modules or some instructions on where to obtain them.

**Comment by DegrizNet** [comment]
Yes I think modules are no longer available. I will try and add as many as i could. 

edit: some modules added! Please let me know if some things are still missing.

**Comment by sreichel** [comment]
Cool stuff. :+1:
